src/config/newGame.ts

```diff
@@ -2,7 +2,7 @@ export const newGameState = () => {
     const gameState: GameState = {
         currentLevel: 1,
         playerBusiness: {
-            name: "Cloud Company",
+            name: null,
```

Comment:
Setting this to null on a brand new game to use as an indicator to eventually start an "intro" level where a user can set their own name. Also let's us know if there is actual user progress yet or not to show the continue button.